rllib/env/remote_base_env.py

```diff
@@ -169,11 +177,13 @@ def poll(
                     self.try_restart(env_id)
                     # Always return multi-agent data.
                     # Set the observation to the exception, no rewards,
-                    # done[__all__]=True (episode will be discarded anyways), no infos.
+                    # terminated[__all__]=True (episode will be discarded anyways),
+                    # no infos.
                     ret = (
                         e,
                         {},
                         {"__all__": True},
+                        {"__all__": False},
```

Comment:
If exception: terminated=True, truncated=False. But it's not really relevant, only that the episode is somewhat done.

Reply:
yep that is correct